### DISCUSSION TITLE
WAHA limit messages

### DISCUSSION THREAD

**Discussion by graficar** [devlikeapro/waha#442] [General]
Good day,

My question is if there is any limit on messages per minute or day to different users?

For example, I have a list of contacts and I need to send a notification to everyone. Could there be any complications with sending mass messages?

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Hi! There's no limits in the project, but you can get ban rely quickly if any of the recipients mark your message as spam
Follow the guide and "act like a human" and it should be good! https://waha.devlike.pro/docs/overview/how-to-avoid-blocking/
Act like a human means:
1. Type before sending message
2. Have a delay
3. send a message
4. Have a delay before sending a new message (like don't send it to 1000 users during 1 minute, it does like suspicious)

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
